Lib/test/test_signal.py

```diff
@@ -698,7 +698,7 @@ def handler(signum, frame):
 @unittest.skipUnless(hasattr(os, "pipe"), "requires os.pipe()")
 class SiginterruptTest(unittest.TestCase):
 
-    def readpipe_interrupted(self, interrupt):
+    def readpipe_interrupted(self, interrupt, timeout):
```

Comment:
In 2 of 3 cases the value is `support.SHORT_TIMEOUT`, so you can make it the default value to reduce the diff.